src/pageComponents/dashboard/cardSection/index.tsx

```diff
@@ -666,8 +784,6 @@ const CardSection = ({
 				return renderMoodboardAndCardType(item);
 			case viewValues.card:
 				return renderMoodboardAndCardType(item);
-			case viewValues.headlines:
-				return renderHeadlinesCard(item);
```

Comment:
Why was this removed?

Reply:
<img width="370" height="1012" alt="Image" src="https://github.com/user-attachments/assets/61210a06-0b9b-4bb0-abda-a932b168c3ea" />

@rogerantony-dev Create a linear issue to remove the headline throughout the codebase? 

This is not a single line change, the issue needs to be created so that we don't forget to clean this up.